File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int j=0,i;
        for( i=0;i<nums.size()-1;i++){
            if(nums[i]!=nums[i+1])
                nums[j++]=nums[i];
            
        }
        nums[j++]=nums[i];
        return j;
    }
};

File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        j=0
        for i in range (0,len(nums)-1):
            if nums[i]!=nums[i+1]:
                nums[j]=nums[i]
                j+=1
        nums[j]=nums[len(nums)-1]
        return j+1